test/net/sourceforge/kolmafia/session/ResultProcessorTest.java

```diff
@@ -90,4 +95,28 @@ public void gettingCosmicBowlingBallInCombatResetsReturnCombats() {
 
     assertThat("cosmicBowlingBallReturnCombats", isSetTo(-1));
   }
+
+  private static AdventureResult GOBLIN_WATER = ItemPool.get(ItemPool.GOBLIN_WATER);
+
+  @Test
+  public void gettingGoblinWaterFromAquagoblinCompletesGoblinQuest() {
+    QuestDatabase.setQuestProgress(QuestDatabase.Quest.GOBLIN, QuestDatabase.STARTED);
+    MonsterData aqua_g_monster = MonsterDatabase.findMonster("Aquagoblin");
```

Comment:
Well, OK, but even in tests, we's prefer AquagoblinMonster

Reply:
changed